### DISCUSSION TITLE
OXY

### DISCUSSION THREAD

**Discussion by protection6** [community/community#69365] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I completed a Textile Industry Web App in Flutter. During the Project I need help at many stages but there was nothing readymade widgets on any web or platform then I decided that I have to complete it in Flutter then It will be my minor contribution to the https://github.com/flutter and https://github.com/fluttercommunity. Now can anyone guide me how can I publish my Project In Flutter or how can I contribute in Flutter?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @protection6! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I do see that the Flutter Community does have [a Contribution Guide](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md) that may be helpful to you to check out.

I've gone ahead and moved this to the correct category for you. Good luck!
